packages/ember-template-compiler/tests/plugins/assert-against-named-outlets-test.js

```diff
@@ -0,0 +1,26 @@
+import { compile } from '../../index';
+import { moduleFor, AbstractTestCase } from 'internal-test-helpers';
+
+moduleFor(
+  'ember-template-compiler: assert-against-named-outlets',
```

Comment:
I'm not sure where to look in CI output to assert this was run on CI.

Reply:
https://github.com/emberjs/ember.js/pull/19691/checks?check_run_id=3214769876#step:8:7625